cmd/cloudx/client/handler.go

```diff
@@ -637,6 +653,11 @@ func (h *CommandHelper) PatchProject(id string, raw []json.RawMessage, add, repl
 	if err != nil {
 		return nil, err
 	}
+
+	if def, _ := h.GetDefaultProjectID(); def == "" {
+		_ = h.SetDefaultProject(id)
+	}
```

Comment:
If I read this correctly, we will set the default project on `ory patch project` if there is no default set yet? Is that not also problematic wrt to side effects?

Reply:
Yeah, I skipped over this in the explanations. It does this also when listing projects. I think by the same reasoning for creating the first project, this is ok.
In my mind setting INVALID --> VALID implicitly is generally ok and convenient.

For listing projects maybe we should only do this, when there is only a single project? Opinions?

Reply:
I think we can remove it completely. In the documentation, we will have to use `--use-project` everywhere anyways, as we don't know whether it's the first or second project the user creates.

I would not expect that patch/update/get/list would change the default value when using a CLI

Reply:
agreed